src/libraries/Microsoft.PowerFx.Core/Public/DependencyFinder.cs

```diff
@@ -31,6 +31,11 @@ public override void Visit(FirstNameNode node)
         {
             var info = _binding.GetInfo(node);
 
+            if (info == null)
+            {
+                return;
+            }
```

Comment:
what's this for?

Reply:
When we have a FirstNameNode which is an identifier, its ocrresponding FirstNameInfo is null, so this is to avoid a crash.

Reply:
Is this still true? it looks like you're setting a first name info on binder.cs:5020